site/content/entry/yuzu-progress-report-may-2022/index.md

```diff
@@ -0,0 +1,235 @@
+
+{{< single-title-imgs-compare
+    "Netto-kun. Now I want a Battle Network Legacy Collection, c'mon CAPCOM!"
+    "./netbug.png"
+    "./netfix.png"
+>}}
+
+yuzu’s icon in the About dialog has been in a struggle lately, a fix for Linux builds would break Windows support, and vice versa.
+Via qtcreator, Docteh {{< gh-hovercard "8339" "fixed the About dialog UI file," >}} and removed an old warning caused by the original .png image.
+Thanks for taking the time to properly address the issue once and for all!
+
+## Controller changes
+
+Motion continued reporting data when disabled, causing `Pokémon Let’s Go, Eevee/Pikachu!` to spam `StopSixAxisSensor` errors in the log.
+While working on this, [german77](https://github.com/german77) noticed a missing parameter, `delta_time`.
+Its implementation allows yuzu to have an {{< gh-hovercard "8308" "accurate motion refresh rate," >}} equal to the Switch.
+
+In an all-in-one pull request, german77 made {{< gh-hovercard "8368" "several input changes," >}} including:
+
+- Add proper error handling for several [HID](https://en.wikipedia.org/wiki/Human_interface_device) functions
+- Improve previous implementations to match more closely to native hardware.
+- Implement functions needed by `Nintendo Switch Sports`, `EnableSixAxisSensorUnalteredPassthrough`, `IsSixAxisSensorUnalteredPassthroughEnabled`, `LoadSixAxisSensorCalibrationParameter`, `GetSixAxisSensorIcInformation`, `ResetIsSixAxisSensorDeviceNewlyAssigned`.
+
+Nintendo Switch Sports will only be playable after an audio rewrite and more GPU work.
+
+`Arcaea` was reported as having issues with touch emulation.
+Turns out this game checks the position on release and some input drivers didn't keep it's position on release. 
+Also, multi-touch wasn't working on touch screens.
+
+After performing basically {{< gh-hovercard "8372" "a mini-rewrite of the touch emulation," >}} german77 fixed both issues.
+
+{{< imgs
+    "./arcaea.png| Osu! but better? (Arcaea)"
+  >}}
+
+One of the hurdles when working with an infinite amount of different controllers is the different quality of implementations they have.
+Since yuzu used to wait for the controller to respond after sending a vibration signal, slow controllers could stall the whole emulator, causing severe stuttering.
+To counter this, german77 {{< gh-hovercard "8374" "moved vibration to a queue in a separate thread," >}} allowing yuzu to move along with emulation while our controllers try to do their best.
+An example of why more CPU threads help more than few faster threads, there’s always some background process that could be made asynchronous.
+
+## Future projects
+
+[Blinkhawk](https://github.com/FernandoS27) has been waiting for NVflinger to be back on track before continuing with `Project Y.F.C.`, so work can resume now.
```

Comment:
Reflect the present -- "While YFC was slightly stalled due to some NVFlinger regressions, these have since been resolved and will be covered in the next Progress Report! YFC is making great progress and is on track to release soon. As a reminder, YFC is an overhaul of various parts of our GPU emulation, fixing many inaccuracies and improving performance."

(or something like that, we should stop assuming every reader knows our silly project names)